scripting/include/srccoop/classdef.inc

```diff
@@ -383,6 +440,27 @@ methodmap CBaseEntity
 	{
 		SetEntProp(this.GetEntIndex(), Prop_Data, "m_spawnflags", iSpawnFlags);
 	}
+	public void GetEyePosition(float vecBuffer[3])
```

Comment:
I thought about leaving it in `CBaseCombatCharacter` but the game defines this function as a virtual `CBaseEntity::GetEyePosition` and it is used in `CBaseEntity::FVisible` directly without a player check. I know SourcePawn doesn't have the concept of traits like other language so that is why I am doing a "virtual" dispatch in `CBaseEntity`.